Userland/Utilities/tail.cpp

```diff
@@ -1,12 +1,13 @@
 /*
- * Copyright (c) 2018-2020, Andreas Kling <kling@serenityos.org>
+ * Copyright (c) 2018-2022, Andreas Kling <kling@serenityos.org>
```

Comment:
We do not update others copyright statements :). You are free to add your own as stated in the [Contribution guidelines](https://github.com/SerenityOS/serenity/blob/master/CONTRIBUTING.md#code-submission-policy)
> Add your personal copyright line to files when making substantive changes. (Optional but encouraged!)